src/main/java/com/everest/InputForParkVehicle.java

```diff
@@ -0,0 +1,31 @@
+package com.everest;
+
+import com.everest.parkinglot.ParkingLot;
+import com.everest.vehicle.Bike;
+import com.everest.vehicle.Car;
+import com.everest.vehicle.Truck;
+import com.everest.vehicle.Vehicle;
+
+public class InputForParkVehicle {
+    Vehicle vehicle;
+    public void parkVehicleInput(String input, ParkingLot parkingLot){
+        String parkingInput[]=input.split(" ");
+        String vehicle_type=parkingInput[0];
+        String registrationNumber=parkingInput[1];
+        String color=parkingInput[2];
+        vehicle = parkVehicleForVehicleType(vehicle_type, registrationNumber, color);
+        System.out.println(parkingLot.parkVehicle(vehicle,parkingLot));
+    }
+
+    public Vehicle parkVehicleForVehicleType(String vehicle_type,String registrationNumber,String color) {
+        switch (vehicle_type){
```

Comment:
SWITCH statement is not preferable for this scenario. Please check Strategy Pattern